src/main/java/com/invicta/impl/hack/hacks/player/SprintHack.java

```diff
@@ -0,0 +1,25 @@
+package com.invicta.impl.hack.hacks.player;
+
+import com.invicta.api.hack.AbstractHack;
+import com.invicta.api.hack.HackCategory;
+
+/**
+ * @author Darki 25/Dec/2019
+ */
+
+@AbstractHack.Hack(name = "Sprint", description = "Makes you sprint!", category = HackCategory.COMBAT)
+public class SprintHack extends AbstractHack {
+	
+	
+	@Override
+	public void onUpdate() {
```

Comment:
There is currently no onUpdate method 😳 I will need to add this sometime when I can be bothered

Reply:
yes yes fix 😳

Reply:
this is meant as an example/testing module anyway. Once you implemented onUpdate you can check if it works using this module